prometheus_all/resources.tf

```diff
@@ -8,12 +8,11 @@ module paas_prometheus_exporter {
   paas_password            = var.paas_exporter_password
 }
 
-module "redis_prometheus_exporter" {
-  source                    = "../redis_prometheus_exporter"
-  count                     = var.redis_service_instance_id == null ? 0 : 1
-  monitoring_instance_name  = var.monitoring_instance_name
-  monitoring_space_id       = data.cloudfoundry_space.monitoring.id
-  redis_service_instance_id = var.redis_service_instance_id
+module "redis_prometheus_exporters" {
+  source                 = "../redis_prometheus_exporter"
+  monitoring_space_id    = data.cloudfoundry_space.monitoring.id
+  count                  = length(var.redis_services)
```

Comment:
Can we change it to a `for_each`? This will be easier to identify the exporters.
ie `redis_prometheus_exporters[3].cloudfoundry_app.redis-exporter` will be `redis_prometheus_exporters["teaching-vacancies-production/teaching-vacancies-redis-queue-production"].cloudfoundry_app.redis-exporter`

Reply:
Then you would have:
`redis_service_instance = each.key`